### ISSUE TITLE
Addition of 'slum'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#41]
@pbuttigieg @rlwalls2008 
Will PCO or ENVO take responsibility for this?

There will probably be crossover with 'informal settlement' and 'inadequate housing', but need SME feedback. 

A household-level definition for 'slum' found in several UN sources. Two are:
http://www.communitascoalition.org/pdf/Communitas_SDSN_Indicators.pdf
http://mdgs.un.org/unsd/mdg/Metadata.aspx?IndicatorId=0&SeriesId=711

A slum household is a household that lacks any one of the following five elements:
- Access to improved water
- Access to improved sanitation
- Sufficient-living area
- Durability of housing
- Security of tenure

**Comment by pbuttigieg** [comment]
ENVO can handle this. I'll add it now.

**Comment by pbuttigieg** [comment]
Present in envo-edit.owl, will be released soon.

**Comment by pbuttigieg** [comment]
I can also add the sanitation and water distribution systems to ENVO.

**Comment by pbuttigieg** [comment]
@rlwalls2008 The population of humans living in a slum sounds PCOish yes?

**Comment by rlwalls2008** [comment]
Yes, it could go in PCO. Do we need a term for a population of people living in a slum? If so, I am happy to add it.

Can also add a term for slum household, if we need that (sibling to agricultural household)

**Comment by pdez90** [comment]
Hi, the word 'slum' appears in the SDG

Indicator 11.1.1: Proportion of urban population living in slums, informal settlement, or inadequate housing

I'm not sure if you think this merits the addition of a term for a population of people living in a slum?

The term 'slum' is one of UNEP's priority terms.

**Comment by mark-jensen** [comment]
@pdez90 The term 'slum' is in ENVO, but isn't currently appearing in SDGIO because the import files need to be updated. The current build was made before 'slum' was added to [envo_terms.txt](https://github.com/SDG-InterfaceOntology/sdgio/blob/master/src/imports/envo_terms.txt)

Either @pbuttigieg or @cmungall will need to rebuild the import files, and maybe create a new release of SDGIO.

> I'm not sure if you think this merits the addition of a term for a population of people living in a slum?

@rlwalls2008 has offered to add it to PCO if needed. But the indicator refers to "urban population living in slums, informal settlement, or inadequate housing". So, would there one term that mentions all locations, or three separate terms, one for each living location?

**Comment by pbuttigieg** [comment]
@pdez90 @mark-jensen @cmungall 
I regenerated the imports in https://github.com/SDG-InterfaceOntology/sdgio/commit/ff69d83758d7f831e0966c0396291187f2c0f17f. 
sdgio-edit.owl not has `slum

**Comment by pbuttigieg** [comment]
>@rlwalls2008 has offered to add it to PCO if needed. But the indicator refers to "urban population living in slums, informal settlement, or inadequate housing". So, would there one term that mentions all locations, or three separate terms, one for each living location? 

I would think that all terms would be useful. The urban population would include slum-dwelling humans, but having a 'slum population' class would be very useful. Yes, we could approach this with axiomatisation, but I feel it would be wise to keep things a little expanded on the class level in the SDGIO case - our users would probably be more comfortable handling classes than alternate axiomatisations.

**Comment by cmungall** [comment]
Is PCO the home for any subclass of population axiomatized using a class from another ontology? E.g. population of humans in a marching band, population of humans in a football team, population of humans in a football team in a bar after a game, ...

**Comment by pbuttigieg** [comment]
Feels a bit like ENVO habitat : there could easily be thousands of terms
which should be kept in a separate artifact, allowing PCO itself to focus
on the core semantics

**Comment by pbuttigieg** [comment]
@mark-jensen @pdez90 
Just made a [new release](https://github.com/SDG-InterfaceOntology/sdgio/tree/v2016-01-22). Terms imported up to today should be included in sdgio.owl.

**Comment by pbuttigieg** [comment]
Note new class `illegal informal settlement` in https://github.com/EnvironmentOntology/envo/issues/265#issuecomment-185779047. Please close this issue if NTR is addressed.
